convert/src/vertex_to_bd/badgerdoc_format/badgerdoc_format.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+from typing import Optional
+
+from ..models.bd_annotation_model import BadgerdocAnnotation
+from ..models.bd_tokens_model import Page
+from ..models.vertex_models import LabelStudioModel
+from .annotation_converter import AnnotationConverter
+from .pdf_renderer import PDFRenderer
+from .plain_text_converter import PlainTextToBadgerdocTokenConverter
+
+
+class BadgerdocFormat:
+    def __init__(
+        self,
+        page_width=595,
+        page_border_offset=15,
+        font_height=11,
+        font_width=7,
+        line_spacing=2,
+    ) -> None:
+        self.page_width = page_width
+        self.page_border_offset = page_border_offset
+        self.font_height = font_height
+        self.font_width = font_width
+        self.line_spacing = line_spacing
+
+        self.tokens_page: Optional[Page] = None
+        self.badgerdoc_annotation: Optional[BadgerdocAnnotation] = None
+        self.pdf_renderer: Optional[PDFRenderer] = PDFRenderer()
+        self.text_converter = PlainTextToBadgerdocTokenConverter()
+
+    def convert_from_labelstudio(self, labelstudio_data: LabelStudioModel):
```

Comment:
```suggestion
    def from_labelstudio(self, labelstudio_data: LabelStudioModel):
```
And one small question - should we really modify class' object that invoked it? From my point of view, functions which are called "converters" (from, to) suppose to return new object of this class. Here, if you really want to modify existing object, you should name it like `update_from_...`